### ISSUE TITLE
Way to show all English decks.

### ISSUE THREAD

**Issue by CookieCoder15** [Lattyware/manydecks#11]
## Describe the feature
Currently, when searching by language, Many Decks will differentiate between different countries. This means that there is and option to search for English (no country), EN-AU, EN-US and so on. It may be good if searching for English decks would return English decks from all countries by default and the user would have to specify if they only wanted from a certain country, as most of the time the country does not matter provided it is the same language.

**Comment by Lattyware** [comment]
Yeah, there are fundamentally different relationships with language variants—some languages like English have a lot of variants where a speaker of one will definitely recognise the other, but some languages have regional variants that, despite a shared root language, aren't really compatible.

This means that root languages have fundamentally different meanings: `en` for example tends to imply "Some form of English" and could encompass the other variants as you suggest, but `pt` refers to Portuguese as spoken in Portugal, and it wouldn't really make sense to include `pt-BR` (Brazilian Portuguese) decks.

In an ideal world, we'd have a complete mapping of say `pt` really implying `pt-PT`, and that `en` is basically a category, etc—but that data doesn't really seem to exist. We are using the IETF Language Subtag Registry for the data we have, which just doesn't contain that level of detail.

Given that, I think there isn't really a one-size fits all answer, so probably the answer is to offer both "any variant of this language" and "specifically only decks that don't specify a region" options and let the user pick what works for them.
